### PR TITLE
Allow setting of command and args on elastalert deployment

### PR DESCRIPTION
**Pull request by brosander**
**What this PR does / why we need it**: Allowing for custom commands and args will give more flexibility to the helm chart. :smile:

I hope to use it with https://github.com/jertel/elastalert-docker/pull/1 but it should be pretty generically useful.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/elastalert/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: ElastAlert is a simple framework for alerting on anomalies, spikes, or other patterns of interest from data in Elasticsearch.
 name: elastalert
-version: 0.2.0
+version: 0.2.1
```
Comment (rjkernick) [inline stable/elastalert/Chart.yaml]: Please bump the minor version as this adds a new feature

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by brosander** [comment]
@rjkernick bumped minor version, rebased on master

thanks for feedback :smile:

**Comment by brosander** [comment]
@rjkernick I rebased on master to remove a conflict in Chart.yaml

**Comment by brosander** [comment]
/assign @prydonius

@prydonius the CI robot suggested you :smile:

**Comment by unguiculus** [comment]
cc @jertel @pickledrick

**Comment by brosander** [comment]
@pickledrick thoughts?

**Comment by brosander** [comment]
@unguiculus @rjkernick @jertel 

Is there someone else I should tag on this?

**Comment by jertel** [comment]
Not that I'm aware of. The original chart author, pickledrick, hasn't chimed in on the past several PRs.

**Pull request merged**